src/fsi/fsi.targets

```diff
@@ -46,9 +46,9 @@
   </ItemGroup>
 
   <ItemGroup>
-    <ProjectReference Include="$(MSBuildThisFileDirectory)..\FSharp.Core\FSharp.Core.fsproj" />
-    <ProjectReference Include="$(MSBuildThisFileDirectory)..\Compiler\FSharp.Compiler.Service.fsproj" />
-    <ProjectReference Include="$(MSBuildThisFileDirectory)..\FSharp.Compiler.Interactive.Settings\FSharp.Compiler.Interactive.Settings.fsproj" />
+    <ProjectReference Include="$(MSBuildThisFileDirectory)../FSharp.Core/FSharp.Core.fsproj" AdditionalProperties="TargetFramework=netstandard2.0" />
```

Comment:
Isn’t using Windows style directory a custom when authoring MSBuild?

Reply:
Not really, we use both, mostly depends on whose typing it in.  Whenever I think about though I go unix style.

Reply:
In the examples above I copied and pasted from the console which is why they are windows style.

Reply:
Make sense. I understand that mostly that's not important, but that's what my observations so that's why I have a question.
Can I assume that this change affects `dotnet build` as well, correct?

And is there chance to use custom build FSI inside Visual Studio? Right now it's still using .NET SDK version

Reply:
It does.

Reply:
ooo ... fsc.exe isn't a linux thing ...  need to change that.  I suspect.

Reply:
That's miserable FSharpCompilerPath is VS only need something like it for straight up sdk.

Reply:
Whole MSBuild overly complicated. I think it can be simplified, but I maybe miss some other scenarios which is important internally. There multiple switches which do approximately the same

Reply:
For example https://github.com/dotnet/fsharp/blob/701b0a25eca276ad7f5e3e00305907f00b720c65/src/FSharp.Build/Microsoft.FSharp.NetSdk.props#L60-L68 may helps with Win/Max/Linux things, but why it's gated behind `DisableAutoSetFscCompilerPath` and what's implications ?